week-2-wide-perspective/w02e05-simple-server-side/src/orders/orders.controller.ts

```diff
@@ -0,0 +1,32 @@
+import { TokenUser } from '../types/user.js'
+import { Router, Request, Response } from 'express'
+import { createOrderValidator } from './orders.validator.js'
+import { getProducts } from '../products/products.service.js'
+import { createOrder, getAllOrders } from './orders.service.js'
+import { validateRequest } from '../validators/validators.service.js'
+
+export const router = Router()
```

Comment:
Tutaj kontrollery są bardzo fajnie opracowane, jedyne co bym zmienił w ich "kreowaniu" to ta nazwa.

`router` w momencie importu trzeba przezywać
A co jakby to tutaj od razu nazwać go `ordersController ` ?:

```
export const ordersController = Router();
```